HnTool/modules/ssh.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, options):
         self.short_name="ssh"
         self.long_name="Checks security problems on sshd config file"
         self.type="config"
-        self.required_files = ['/etc/ssh/sshd_config', '/etc/sshd_config']
```

Comment:
'Cause that config file when does not exist the ssh module not perform and when you create it duplicate the output.